hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -834,6 +834,7 @@ public void testCleaningArchivingAndCompaction() throws Exception {
         .withCompactionConfig(HoodieCompactionConfig.newBuilder().archiveCommitsWith(2, 4)
             .withFailedWritesCleaningPolicy(HoodieFailedWritesCleaningPolicy.NEVER)
             .retainCommits(1).retainFileVersions(1).withAutoClean(true).withAsyncClean(false).build())
+        .withEmbeddedTimelineServerEnabled(false)
```

Comment:
why are we doing this change? this is turned on by default in the real world?

Reply:
yes, will revert before landing